e2e/app/setup.go

```diff
@@ -565,6 +566,7 @@ func writeMonitorConfig(ctx context.Context, def Definition, logCfg log.Config,
 	const (
 		privKeyFile        = "privatekey"
 		xCallerPrivKeyFile = "xcaller_privatekey"
+		flowGenPrivKeyFile = "flowgen_privatekey"
```

Comment:
you need to add this file to `vmcompose#Upgrade` so that is copies the file to VMs when upgrading protected networks

Reply:
Thanks, indeed this was missing.